tests/TestCase/Command/RoutesCommandTest.php

```diff
@@ -189,6 +189,6 @@ public function tearDown(): void
         parent::tearDown();
 
         // remove routes files
-        \array_map('unlink', \glob(CONFIG . 'routes*.php'));
+        \array_map('unlink', \glob(CONFIG . '*routes*.php'));
```

Comment:
why this change? was the glob not working without the leading `*`?

Reply:
yes, the compiled_routes doesn't get deleted

Reply:
there is a file core_routes, i renamed it to routes_core